Implementation of #106 - Siemens Energy Color Scheme

IremToroslu opened the pull request:
#106

Waldleufer commented on frontend/src/components/cards/MiniCardComponent.js:
```diff
@@ -13,7 +13,7 @@ const useStyles = createUseStyles((theme) => ({
         // padding: '8px 16px 8px 16px',
         '&:hover': {
-            borderColor: theme.uniformStyle.color.tableHeaderColor,
+            borderColor: '#AE56FF',
```
please add at least a comment here, or maybe consider creating a variable for this color, because like this its a "Magic value" 🙂

IremToroslu replied:
I am on it 😄

Waldleufer commented on frontend/src/components/details/navbar.css:
```diff
@@ -1,30 +1,90 @@
+}
+
+/* -webkit-radial-gradient(  bottom right, #4D217A,#1B1534 ); */
+:root{
+    --global--button--color:#836387;
+    --global--button--text--color:#F0F0F0;
+    --global--panel-title--FontSize: 22px;
+    --global--text-subtitle--FontSize: 19px;
 }
```
Consider maybe moving the variables to the variables file and adding a comment on what these variables are for?

IremToroslu replied:
I did not do the documentation I will do it today. 😄

Waldleufer reviewed (approved):
It looks very good!

Thank you for also cleaning up most of the css that was scattered around in some places whenever you came in contact with it. That helps with #78 :D 

Some Points: 

- [ ]  The inner background of the Login looks a bit yellowish to me, I find that doesn't go too well with text input fields. Maybe we could make these two colours either match or differ more?
![image](https://user-images.githubusercontent.com/22003632/121817908-7542e600-cc84-11eb-8908-f1b0ae0546b3.png)

- [ ] May I suggest using the same colour that we use in the header also for the navbar in the Details Page? But maybe mirror it, if it is on the compare page the second one? 
![image](https://user-images.githubusercontent.com/22003632/121818007-eaaeb680-cc84-11eb-8b09-b074e7010948.png)

- [ ] for the DetailsPage in Compare mode the PDF Button explodes the bar: 
![image](https://user-images.githubusercontent.com/22003632/121818049-2e092500-cc85-11eb-9e8d-995bb36e926c.png)

IremToroslu commented on frontend/src/components/details/PieChartComponent.jsx:
```diff
@@ -31,17 +31,17 @@ const PieChartComponent = () => {
-            theme.color.StainlessSteel,
+            '#F7EA48',
+            '#CBD2DE',
+            '#FDB0C0',
+            '#A4C8D5',
             theme.color.Alminium,
-            theme.color.SiliconSteel,
-            theme.color.Copper
+            '#52bede',
+            '#D89279'
```
this button sizing issue  needs to be done in #78 right ? 😄

IremToroslu replied:
@gandompm @Waldleufer

IremToroslu commented:
> It looks very good!
> 
> Thank you for also cleaning up most of the css that was scattered around in some places whenever you came in contact with it. That helps with #78 :D
> 
> Some Points:
> 
> * [x]   The inner background of the Login looks a bit yellowish to me, I find that doesn't go too well with text input fields. Maybe we could make these two colours either match or differ more?
>   ![image](https://user-images.githubusercontent.com/22003632/121817908-7542e600-cc84-11eb-8908-f1b0ae0546b3.png)
> * [x]  May I suggest using the same colour that we use in the header also for the navbar in the Details Page? But maybe mirror it, if it is on the compare page the second one?
>   ![image](https://user-images.githubusercontent.com/22003632/121818007-eaaeb680-cc84-11eb-8b09-b074e7010948.png)
> * [ ]  for the DetailsPage in Compare mode the PDF Button explodes the bar:
>   ![image](https://user-images.githubusercontent.com/22003632/121818049-2e092500-cc85-11eb-9e8d-995bb36e926c.png)

I changed the navbar color again and also fixed the pdf button til it reaches %125 zoom at then after zooming it breaks and when it comes to mobile view it also breaks as well. We need to work on that

IremToroslu commented:
Thanks @Waldleufer 😄 for the feedback, appreciate it :)

IremToroslu commented:
![image](https://user-images.githubusercontent.com/29985277/121912272-be507400-cd30-11eb-812c-7939c80eb2b5.png)
![image](https://user-images.githubusercontent.com/29985277/121912373-d32d0780-cd30-11eb-8f27-275d9d266b06.png)


Pls kindly find the new look and feel of the Carbon foot print visualization site 😄 👣

MartinDuersch reviewed (approved):
New Color Scheme looks good!

SaiVarunVaranasi reviewed (commented):
looks good :-)
